installer/bundle/bundle_skel_PPC-RHEL.sh

```diff
@@ -0,0 +1,584 @@
+#!/bin/sh
+
+#
+# Shell Bundle installer package for the SCX project
+#
+
+PATH=/usr/bin:/usr/sbin:/bin:/sbin
+umask 022
+
+# Can't use something like 'readlink -e $0' because that doesn't work everywhere
+# And HP doesn't define $PWD in a sudo environment, so we define our own
+case $0 in
+    /*|~*)
+        SCRIPT_INDIRECT="`dirname $0`"
+        ;;
+    *)
+        PWD="`pwd`"
+        SCRIPT_INDIRECT="`dirname $PWD/$0`"
+        ;;
+esac
+
+SCRIPT_DIR="`(cd \"$SCRIPT_INDIRECT\"; pwd -P)`"
+SCRIPT="$SCRIPT_DIR/`basename $0`"
+EXTRACT_DIR="`pwd -P`/scxbundle.$$"
+
+# These symbols will get replaced during the bundle creation process.
+#
+# The OM_PKG symbol should contain something like:
+#       scx-1.5.1-115.rhel.6.ppc (script adds .rpm)
+# Note that for non-Linux platforms, this symbol should contain full filename.
+#
+# PROVIDER_ONLY is normally set to '0'. Set to non-zero if you wish to build a
+# version of SCX that is only the provider (no OMI, no bundled packages). This
+# essentially provides a "scx-cimprov" type package if just the provider alone
+# must be included as part of some other package.
+
+TAR_FILE=<TAR_FILE>
+OM_PKG=<OM_PKG>
+OMI_PKG=<OMI_PKG>
+PROVIDER_ONLY=0
+
+SCRIPT_LEN=<SCRIPT_LEN>
+SCRIPT_LEN_PLUS_ONE=<SCRIPT_LEN+1>
+
+# Packages to be installed are collected in this variable and are installed together 
+ADD_PKG_QUEUE=
+
+# Packages to be updated are collected in this variable and are updated together 
+UPD_PKG_QUEUE=
+
+usage()
+{
+    echo "usage: $1 [OPTIONS]"
+    echo "Options:"
+    echo "  --extract              Extract contents and exit."
+    echo "  --force                Force upgrade (override version checks)."
+    echo "  --install              Install the package from the system."
+    echo "  --purge                Uninstall the package and remove all related data."
+    echo "  --remove               Uninstall the package from the system."
+    echo "  --restart-deps         Reconfigure and restart dependent service"
+    echo "  --source-references    Show source code reference hashes."
+    echo "  --upgrade              Upgrade the package in the system."
+    echo "  --version              Version of this shell bundle."
+    echo "  --version-check        Check versions already installed to see if upgradable"
+    echo "                         (Linux platforms only)."
+    echo "  --debug                use shell debug mode."
+    echo "  -? | --help            shows this usage text."
+}
+
+source_references()
+{
+    cat <<EOF
+-- Source code references --
+EOF
+}
```

Comment:
Did you verify that --source-references actually works when you build a kit? It probably does, but it's fast to verify and something easy to miss ...